boards/shared/Src/Io/Io_SharedErrorTable.c

```diff
@@ -52,6 +52,9 @@ static void Io_ProcessDimNonCriticalErrorMsg(
     SET_ERROR(
         error_table, DIM_NON_CRITICAL_STACK_WATERMARK_ABOVE_THRESHOLD_TASK1HZ,
         data->stack_watermark_above_threshold_task1_hz);
+    SET_ERROR(
```

Comment:
Similar error to the one mentioned [here](https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/642/files#r436229383), except this time it's for `Io` layer